app/_src/gateway/upgrade/lts-upgrade.md

```diff
@@ -0,0 +1,217 @@
+1. Review the gateway changes from 2.8 to 3.4 for any breaking changes that affect your deployments.
+1. Conduct a thorough examination of the modifications made to the `kong.conf` file between the LTS releases.
+1. Using your chosen strategy, test migration in a pre-production environment.
+
+**Performing the upgrade**
+
+The actual execution of the migration is dependent on the type of deployment you have with Kong. 
+In this part of the upgrade journey, you will use the strategy you determined during the preparation phase.
+
+1. Execute your chosen upgrade strategy on dev.
+2. Move from dev to prod.
+3. Smoke test.
+4. Wrap up the upgrade or roll back and try again.
+
+Now, let's move on to preparation, starting with your backup options.
+
+## Prepation: Choosing a backup strategy
+
+The following instructions lay out how to back up your configuration for each supported deployment type. This is an important step prior to upgrading. Each supported deployment mode has different instructions for backup.
+
+The `kong migrations` commands in this guide are not reversible. We recommend backing up data before any migration. 
+
+* **Database backup** (_Traditional mode and control planes in hybrid mode_): PostgreSQL has native exporting and importing tools that are reliable and performant, and that ensure consistency when backing up or restoring data.
+* **Declarative backup** (_DB-less mode and data planes in hybrid mode_): In DB-less mode, configuration is managed declaratively using a tool called decK. decK allows you to import and export configuration using YAML or JSON files. 
+
+Review the [Backup and Restore](/gateway/{{page.kong_version}}/upgrade/backup-and-restore/) guide to prepare backups of your configuration. If you run into any issues and need to roll back, you can also reference that guide to restore your old data store.
+
+## Preparation: Choosing an upgrade strategy
+
+Upgrade strategies introduced in this section are generic and may or may not fit in with your deployment environment. 
+
+Choose your deployment mode:
+* [Traditional](#traditional-mode)
+* [Hybrid](#hybrid-mode)
+* [DB-less](#db-less-mode)
+
+Here's a flow chart the summarizes how the decision process works:
+
+![Choosing a strategy based on the deployment mode](/assets/images/products/gateway/upgrade/choose-your-deployment.png)
+> _Figure 1: Choosing an upgrade strategy based on deployment type_
+
+See the following sections for breakdowns of each strategy.
+
+### Traditional mode
+
+A traditional mode deployment is when all {{site.base_gateway}} components are running in one environment, and there is no control plane/data plane separation.
+
+You have two options when upgrading {{site.base_gateway}} in traditional mode:
+* **Dual-cluster upgrade**: A new Kong cluster of version Y is deployed alongside the current version X, so that two clusters serve requests concurrently during the upgrade process.
```

Comment:
```suggestion
* **Dual-cluster upgrade**: A new {{site.base_gateway}} cluster of version Y is deployed alongside the current version X, so that two clusters serve requests concurrently during the upgrade process.
```